wire-ios/Wire-iOS/Sources/UserInterface/Components/Giphy/GiphyConfirmationViewController.swift

```diff
@@ -59,11 +59,11 @@ final class GiphyConfirmationViewController: UIViewController {
             imagePreview.animatedImage = previewImage
         }
 
-        let closeImage = StyleKitIcon.cross.makeImage(size: .tiny, color: SemanticColors.Icon.foregroundDefaultBlack)
-        navigationItem.rightBarButtonItem = UIBarButtonItem(image: closeImage,
-                                                            style: .plain,
-                                                            target: self,
-                                                            action: #selector(GiphySearchViewController.onDismiss))
+        let closeItem = UIBarButtonItem.closeButton(action: { [weak self] _ in
+            self?.navigationController?.dismiss(animated: true, completion: nil)
```

Comment:
I think here it should also be:
```suggestion
            self?.presentingViewController?.dismiss(animated: true, completion: nil)
```